src/shared/cgroup-setup.c

```diff
@@ -731,7 +716,15 @@ int cg_trim_everywhere(CGroupMask supported, const char *path, bool delete_root)
         if (q > 0)
                 return r;
 
+        return cg_trim_controllers(supported, _CGROUP_MASK_ALL, path, delete_root);
+}
+
+int cg_trim_controllers(CGroupMask supported, CGroupMask mask, const char *path, bool delete_root) {
```

Comment:
hmm, this call is now cgroupsv1 specific, unlike most other calls here, and there's not even an assert() to check it's not called on the wrong environment... or clarified in the name or so

Reply:
Distinguished the functions by their name.